backends/apple/coreml/partition/coreml_partitioner.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2023 Apple Inc. All rights reserved.
+#
+# Please refer to the license found in the LICENSE file in the root directory of the source tree.
+
+import logging
+from typing import List
+
+import torch
+from torch._export.exported_program import ExportedProgram
+from torch.fx.passes.infra.partitioner import CapabilityBasedPartitioner
+from torch.fx.passes.operator_support import OperatorSupportBase
+
+from executorch.exir.backend.partitioner import (
+    DelegationSpec,
+    Partitioner,
+    PartitionResult,
+)
+from executorch.backends.apple.coreml.compiler.coreml_preprocess import CoreMLBackend
+
+from coremltools.converters.mil.frontend.torch.torch_op_registry import is_torch_fx_node_supported
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+
+class OperatorsSupportedForCoreMLBackend(OperatorSupportBase):
+    def __init__(self, skip_ops: List[str] = []) -> None:
+        super().__init__()
+        self.skip_ops = skip_ops
+
+    def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
+        # get_attr node can always be supported on any backend
+        if node.op == "get_attr":
+            return True
+        # check if the PyTorch op get called is supported in Core ML
+        elif node.op == "call_function":
+            return is_torch_fx_node_supported(node, self.skip_ops)
+        # cowardly refuse to support all other types of node
+        else:
+            return False
```

Comment:
Maybe raise an exception with the op name specified, like

```
raise NotImplementedError(f"operator {node.name} is not supported in CoreMLBackend yet")
```

Reply:
Should we kill the partition with an exception, or return False so it may still get a chance to be executed in other backends?

IIUC, the partitioner works by checking which node is supported by which backend(s). So if Core ML does not support something, we return false then let other backends try?